### PR TITLE
Minor bug fixes

### LINKED ISSUE #183: Tab character is displayed with 'A' in BBEdit
# Environment

```none
Cascadia Code version number: 1910.04 (and with November Update #181 builds)
Application (with version) used to display text: BBEdit 13.0.3
OS platform and version: macOS 10.15.1
Screen resolution (i.e. 220dpi): n/a
```

# Steps to reproduce

1. Download and install BBEdit 13.0.2 from https://www.barebones.com/support/bbedit/updates.html
2. Create a new text document in BBEdit
3. Enter a `tab` character (0x0009)

# Expected behavior

The tab character should appear as ‘clear’. For reference, using Consolas:

consolas-tab-example

# Actual behavior

Using Cascadia Code (including the new November update #181 and the Mono variants) this is displayed thusly:

cascadia-tab-example

Note that the Mono variants display an accented A (Á) in the tab character.

### LINKED ISSUE #186: Odd Padding in VSCode Terminal (Xterm)
# Environment

```none
Cascadia Code version number: 1910.04
Application (with version) used to display text: vscode-insiders and vscode
OS platform and version: Windows 10 GA and Insiders
Screen resolution (i.e. 220dpi): Multiple monitors, multiple DPIs tested

Any other software?
```

# Steps to reproduce
1. Set "terminal.integrated.fontFamily": "Cascadia Code"
2. Open terminal
3. Use some characters that have a background color
4. There will be background padding at the bottom.

# Expected behavior
No additional padding below the lines

# Actual behavior
image
Note the padding at the bottom. No other font tested does this.

### LINKED ISSUE #205: Some more characters missing for codepage 437 and 850 console compatibility
This version does not yet support codepages 437 and 850 completely.

I guess I should have included more details in #142, but the missing characters are not easy to generate with easy to reproduce steps in cmd.exe.

The last missing glyphs are in the characters for values 0x01 to 0x1F, which are typically used as C0 control characters, but can be output as characters by CUI apps that directly control the screen buffer.

CP437-850 missing glyphs

The same characters with Consolas font:
CP437-850 Consolas

So for complete compatibility with older CUI apps, the following characters are also required:

Hex 0123456789ABCDEF
0_ ☺☻♥♦♣♠•◘○◙♂♀♪♫☼
1_ ►◄↕‼¶§▬↨↑↓→←∟↔▲▼

Their Unicode codepoints are as follows:
0_ : (NULL), U+263A, U+263B, U+2665, U+2666, U+2663, U+2660, U+2022, U+25D8, U+25CB, U+25D9, U+2642, U+2640, U+266A, U+266B, U+263C
1_ : U+25BA, U+25C4, U+2195, U+203C, U+00B6, U+00A7, U+25AC, U+21A8, U+2191, U+2193, U+2192, U+2190, U+221F, U+2194, U+25B2, U+25BC

As far as I can tell, the characters in 0x01, 0x02, 0x0E and 0x0F are missing, codepoints U+263A, U+263B, U+266B and U+263C, which are the two smiley faces, the double music note and the sun.
It would be great to have these included for full console compatibility.

### LINKED ISSUE #206: Add Unicode Hyphen U+2010
It seems that the font does not include the U+2010 hyphen character. I believe this is used for hyphens in linux man pages so the hyphens appear as boxes instead. 

It would make linux terminal windows look much better if this character were added.

### PR DESCRIPTION
**Pull request by aaronbell**
## Summary of the Pull Request

This minor update resolves several bugs through the addition of a few missing glyphs, repositioning PL characters for better alignment, and resolving an issue with Apple's ATSUI API. The new glyphs are hinted. 

## PR Checklist
* [x] Closes #183 - uni000D added to help font play nice with ATSUI.
* [x] Closes #205 - smiley faces, music note, and sun added to complete codepage coverage.
* [x] Closes #206 - hyphen character added.
* [x] CLA signed. 
* [ ] I've discussed this with core contributors already.

## Detailed Description of the Pull Request / Additional comments

New characters added:
Screen Shot 2019-12-03 at 9 55 44 PM

### CODE REVIEW

File: sources/CascadiaCode-Regular.ufo/lib.plist [hunk 1]
```diff
@@ -13,9 +13,13 @@
...
     <integer>110</integer>
     <key>com.schriftgestaltung.appVersion</key>
-    <string>1272</string>
+    <string>1274</string>
     <key>com.schriftgestaltung.customParameter.GSFont.DisplayStrings</key>
-    <string></string>
+    <array>
+      <string>/ereversed-cy</string>
+    </array>
+    <key>com.schriftgestaltung.customParameter.GSFont.Make morx table</key>
```
Comment (DHowett-MSFT) [inline sources/CascadiaCode-Regular.ufo/lib.plist]: do we definitely want the `morx` table? this looks like it's setting "make morx table" to `1`

File: sources/nerdfonts/NerdfontsPL.ufo/glyphs/uniE_0B_6.glif [hunk 2]
```diff
@@ -7,26 +7,20 @@ uniE0B6
...
-      <point x="805" y="-430"/>
-      <point x="1003" y="-492"/>
+      <point x="1200" y="-692" type="curve"/>
+      <point x="1200" y="-692" type="line"/>
+      <point x="1200" y="1790" type="line"/>
+      <point x="477" y="1790"/>
+      <point x="0" y="1297"/>
+      <point x="0" y="549" type="curve" smooth="yes"/>
+      <point x="0" y="-199"/>
+      <point x="477" y="-692"/>
```
Comment (DHowett-MSFT) [inline sources/nerdfonts/NerdfontsPL.ufo/glyphs/uniE_0B_6.glif]: wow, this glyph got way simpler

**Comment by DHowett-MSFT** [comment]
(i do, however, _love_ the smiley faces. they've got personality!)

File: sources/CascadiaCode-Regular.ufo/lib.plist (hunk 1 above)
Comment (aaronbell) [reply sources/CascadiaCode-Regular.ufo/lib.plist]: Derp. No. That setting actually doesn't do anything (apparently you have to *have* a <morx> table in order for it to work), and I forgot to remove it.

File: sources/nerdfonts/NerdfontsPL.ufo/glyphs/uniE_0B_6.glif (hunk 2 above)
Comment (aaronbell) [reply sources/nerdfonts/NerdfontsPL.ufo/glyphs/uniE_0B_6.glif]: Yup. On this one and the one facing the other way, I removed some of the intermediary points, which were unnecessary holdovers from the TTF source file. It'd be nice to go through and clean up all the Nerd Fonts / PowerLine symbols, but that'd be quite a task.

**Comment by aaronbell** [comment]
That's super strange about the Powerline symbols floating too low in your install. They were working perfectly in VS Code. It might be an issue with how different applications render the character space. I also note that your version has a lot less space on the bottom than mine. 

Screen Shot 2019-12-03 at 11 12 27 PM

**Comment by aaronbell** [comment]
Unfortunately, I don't have the time at the moment to sort this issue out, so am going to close the PR and re-submit soon!

**Pull request closed without merging**